internal/archive/archive.go

```diff
@@ -42,6 +43,12 @@ func Open(options *Options) (Archive, error) {
 	if err != nil {
 		return nil, err
 	}
+	if options.Pro != "" {
+		err = validatePro(options.Pro)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
I would prefer to inline this, it is more clear in my opinion.

Reply:
Removed this altogether due to my comments about not allowing other unknown `pro` values.